crates/swc_plugin_runner/src/wasix_runtime.rs

```diff
@@ -68,7 +68,7 @@ pub fn build_wasi_runtime(
 
     let dummy_loader = BuiltinPackageLoader::new_with_client(".", Arc::new(StubHttpClient));
     let rt = PluggableRuntime {
-        rt: Arc::new(TokioTaskManager::shared()),
+        rt: Arc::new(TokioTaskManager::default()),
```

Comment:
shared tokio runtime was removed: https://github.com/wasmerio/wasmer/commit/1f644f847a2e40c94cb6dd9ab5cd7e635e62657a

Reply:
Yes, and need equivalent instead of just using default.

Reply:
I couldnt find any official way to use shared runtime. It seems they started using pooling pattern. I dont know this codebase yet well enough to do the appropriate change.

Reply:
Well, we can't just switch to default runtime in any case. One of the reason I didn't upgrade wasmer yet.

Reply:
Ye it doesnt work like this